test/e2e-go/restAPI/restClient_test.go

```diff
@@ -146,7 +146,7 @@ func waitForRoundOne(t *testing.T, testClient libgoal.Client) {
 	select {
 	case err := <-errchan:
 		require.NoError(t, err)
-	case <-time.After(1 * time.Minute): // Wait 1 minute (same as WaitForRound)
+	case <-time.After(2 * time.Minute): // Wait 2 minutes (same as WaitForRound)
```

Comment:
why ? The goal of this timeout was to be 100% aligned with the one in WaitForRound in the algod.
If a test you're running is failing, you should find out why it is failing.

Reply:
Thanks, I changed it back